capella_model_explorer/app.py

```diff
@@ -30,10 +31,13 @@
 logger.info("\tRoute prefix: '%s'", c.ROUTE_PREFIX)
 logger.info("\tLive mode: %s", c.LIVE_MODE)
 logger.info("\tHost: '%s'", c.HOST)
-logger.info("\tRoute prefix: '%s'", c.ROUTE_PREFIX)
```

Comment:
We should keep this log. It's still part of the app configuration, and could become helpful for debugging connectivity issues. Besides, there's no redirect from anywhere other than `/`, so e.g. a misconfiguration that causes a wrong prefix to be used downstream would just result in a lot of 404s.